lib/airbrake/rack/user.rb

```diff
@@ -50,7 +50,9 @@ def as_json
       private
 
       def try_to_get(key)
-        String(@user.__send__(key)) if @user.respond_to?(key)
+        String(@user.__send__(key)) if @user.respond_to?(key) &&
+                                       (@user.method(key).arity.zero? ||
+                                        @user.method(key).arity == -1)
```

Comment:
comparison with <= 0 will not work, because methods with variable number of arguments, that have also required arguments will return < -1